packages/suite/src/utils/firmware/index.ts

```diff
@@ -109,3 +118,11 @@ export const validateFirmware = (
         return 'TR_FIRMWARE_VALIDATION_T1_V2';
     }
 };
+
+export const getFirmwareType = (device?: TrezorDevice) => {
```

Comment:
just curious, what's the reason for this and `isBitcoinOnlyFirmware` being in separate packages?

Reply:
`hasBitcoinOnlyFirmware` can be reused in different packages

`getFirmwareType` is for translating `Regular` to `Universal` in Suite. 
Answer ⬇️